docs/FAQ.md

```diff
@@ -5,3 +5,37 @@ This FAQ is in progress. If there is anything you'd like to see added, please re
 ##### What is the difference between cron, batchtime, and periodic build?
 
 The main difference is that cron and batchtime are tied to activating builds for existing commits and periodic builds creates new builds on a schedule regardless of commit activity. For more on their differences and examples, see [controlling when tasks run](Project-Configuration/Controlling-when-tasks-run).
+
+#### Dynamic Github Access Tokens
+
+#### Can more than one app be associated with my project?
+
+No, only one app per project is supported.
+
+#### Why am I seeing `HTTP status code 401: 401 (Unauthorized): requester does not have permission to create a token` error when I try to clone with a generated token?
+
+It is possible that the token is being generated with permissions being fully restricted to `No Permissions` in the project settings. Double check that the [project settings](Github-Integrations#dynamic-github-access-tokens) did not restrict this requester to `No Permissions`.
+
+#### Why am I seeing a `Remote: Repository not found` error when I try to clone with a generated token?
+
+It is possible that either the GitHub app associated with the project does not have permission to clone this repo, or the token is being generated with _restricted permissions_. Double check that _both_ the project settings and the command, if they specify restricted permissions, include `contents:read`.
+
+#### Why am I seeing `the intersection of the project setting's requester permissions and provided permissions does not have any permissions to create a token`?
+
+The token restrictions in the project settings and those specified in the command can intersect in a way that results in the token having no restrictions. For example, if the project settings specifies `{ checks: read }` for the requester, a token generated by that requester will have `{ checks: read }` permissions and nothing else. Therefore, if the command specifies `{ actions: read }` for example, `actions:read` will not be added because project settings restricts it to only `checks:read`, and because the command is restricting it to only `actions:read`, the `checks:read` from the project is now also not available, leaving it with no permissions and that error. For more information on this, see [project settings](Github-Integrations#dynamic-github-access-tokens).
+
+#### Why am I seeing `remote: Invalid username or password fatal: Authentication failed for` when I try use a generated token?
+
+It is possible that your token has expired. Make sure that the token was not generated more than an hour before use.
```

Comment:
It's also possible for the token to not have permission for that repository! i.e. the github app doesn't